components/lists/FeedList.tsx

```diff
@@ -1,15 +1,128 @@
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { useTheme } from '@react-navigation/native';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import { Image } from 'expo-image';
+import { Link } from 'expo-router';
 import React from 'react';
-import { View } from 'react-native';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { blurhash } from '../../config/constants';
+import { Feed as FeedData } from '../../mock/feed';
+import Row from '../common/Row';
+import StyledText from '../common/StyledText';
 
-// TODO: - Implement a feed list-view of feeds as seen in the design
-/**
- * - order the feeds to most recent first
- * - clicking on the share button on each feed should open the bottom-sheet and pass params to the bottom-sheet - only implement this part if you’re comfortable working with bottomsheet
- * - add tests for the feed list component
- */
+dayjs.extend(relativeTime);
+
+interface FeedListItemProps {
+  item: Feed;
+}
+
+type Feed = {
+  title: string;
+  body: string;
+  topic: string;
+  url: string;
+  image: string;
+  created_at: string;
+};
+
+const { height } = Dimensions.get('screen');
+
+const FeedListItem = ({ item }: FeedListItemProps) => {
+  const { colors } = useTheme();
+  return (
+    <View style={[styles.feed, { borderBottomColor: colors.border }]} testID="feedItem">
+      <StyledText size="md" font="regular">
+        {item.body}
+      </StyledText>
+      <View style={styles.imageContainer}>
+        <Image
+          style={styles.image}
+          source={{ uri: item.image }}
+          transition={1000}
+          contentFit="cover"
+          placeholder={blurhash}
+        />
+      </View>
+
+      <Row>
+        <Link
+          href={{
+            pathname: '/home/feed/share',
+          }}
+        >
+          <View style={styles.share}>
+            <StyledText size="md" allowFontScaling font="bold" variant="link" style={styles.shareLabel}>
+              Share
+            </StyledText>
+            <MaterialCommunityIcons size={20} name="share" color={colors.primary} />
+          </View>
+        </Link>
+
+        <StyledText size="sm">{dayjs(item.created_at).fromNow()}</StyledText>
+      </Row>
+    </View>
+  );
+};
 
 const FeedList = () => {
-  return <View />;
+  const recentFirst = (data: Array<Feed>) => {
+    return data.sort((a, b) => {
+      const dateA = new Date(a.created_at);
+      const dateB = new Date(b.created_at);
+
+      if (dateA > dateB) {
+        return -1;
+      } else if (dateA < dateB) {
+        return 1;
+      } else {
+        return 0;
+      }
+    });
```

Comment:
We can simplify this to `return data.sort((a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime());`